build/prep-img.sh

```diff
@@ -144,7 +144,7 @@ debug 'config.txt' "`cat "$BOOT_CONFIG_FILE" | grep 'gpio-fan'`"
 echo
 echo "Copying programs.."
 GIT_TMP=`mktemp -d`
-git clone -q git@github.com:pmansour/minimeet.git "$GIT_TMP/minimeet/"
+git clone -q https://github.com/pmansour/minimeet.git "$GIT_TMP/minimeet/"
```

Comment:
this is a great change useful across all OS versions. Do you think you could separate this into its own PR so we can merge it straight away? The other changes might need some more thinking.

Reply:
New PR raised